CMSgov/design-system
# [WNMGDS-907] Add comment to top of ClearIcon files

## Summary
For some reason, adding comments in these files make it so that they are actually processed by the typescript doc generator...

### Added
Comments :shrug:

### Fixed
![image](https://user-images.githubusercontent.com/21293083/118265357-102e8180-b487-11eb-92e3-5c7ba7d320ab.png)

## How to test
- Run `yarn build` on master, notice this error: 
![image](https://user-images.githubusercontent.com/21293083/118265330-04db5600-b487-11eb-9e5d-c0dc87997fe5.png)

- Run on this branch. Notice no error.
